src/init.ts

```diff
@@ -0,0 +1,5 @@
+import Shopify from '@shopify/shopify-api';
+
+export default function initializeShopify(context: typeof Shopify.Context) {
+  Shopify.Context.initialize(context);
+}
```

Comment:
This is an improvement over the previous iteration where we needed to export the `Shopify` object, but I'm not satisfied yet. It still requires apps to do:

```ts
Shopify.Context.initialize({ ... });
initializeShopify(Shopify.Context);
```

but I couldn't think of a nice way of getting rid of the second call, even though it is succinct enough (IMHO).

Reply:
Wait, is `Shopify.Context` being reinitialized? We initialize it, pass it to `initializeShopifyKoa` where it's initialized again as itself? I feel like I remember you explaining why something like this is necessary here but now I can't quite remember why.

Reply:
The first `initialize` call here is in the actual app code, so it sets up its own copy of the library. `koa-shopify-auth` might bring in a different copy of the library which we also need to make sure is initialized, hence the secondary call.